packages/frontend/src/components/AllBids/AllBidsList.tsx

```diff
@@ -18,14 +19,19 @@ export const AllBidsList = ({ search }: AllBidsListProps) => {
     [search]
   )
 
+  const raffleBidsOffset = Math.max(0, bids.length - RAFFLE_PARTICIPANTS_COUNT)
+  const firstRaffleBidIndex =
+    raffleBidsOffset >= AUCTION_PARTICIPANTS_COUNT ? AUCTION_PARTICIPANTS_COUNT : raffleBidsOffset
+
   const auctionBids = useMemo(() => {
-    const sectionBids = bids.length <= AUCTION_PARTICIPANTS_COUNT ? bids : bids.slice(0, AUCTION_PARTICIPANTS_COUNT)
+    const sectionBids = bids.length <= RAFFLE_PARTICIPANTS_COUNT ? [] : bids.slice(0, firstRaffleBidIndex)
     return search ? searchBid(sectionBids) : sectionBids
-  }, [search, bids, searchBid])
+  }, [search, bids, searchBid, firstRaffleBidIndex])
   const raffleBids = useMemo(() => {
-    const sectionBids = bids.length <= AUCTION_PARTICIPANTS_COUNT ? [] : bids.slice(AUCTION_PARTICIPANTS_COUNT)
+    const sectionBids = bids.length > RAFFLE_PARTICIPANTS_COUNT ? bids.slice(firstRaffleBidIndex) : bids
```

Comment:
Can be simplified to `const sectionBids = bids.slice(0, firstRaffleBidIndex)` as in case `bids.length <= RAFFLE_PARTICIPANTS_COUNT` `firstRaffleBidIndex` equals 0.

Reply:
Should be `const sectionBids = bids.slice(firstRaffleBidIndex)` instead of `const sectionBids = bids.slice(0, firstRaffleBidIndex)` but overall you're right. Nice catch 👌